From ed25b0e63b2bedecc9c181a620422627f27064ef Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@hadrons.org>
Date: Tue, 23 Jul 2019 01:28:10 +0200
Subject: [PATCH libaio 12/28] man: Fix title header

- Update year.
- Balance double quotes.
- Remove version from source argument as recommended in man-pages(7).
- Fix all sections numbers to 3.

Signed-off-by: Guillem Jover <guillem@hadrons.org>
Signed-off-by: Jeff Moyer <jmoyer@redhat.com>
---
 man/io.3               | 2 +-
 man/io_cancel.3        | 2 +-
 man/io_fsync.3         | 2 +-
 man/io_getevents.3     | 2 +-
 man/io_prep_fsync.3    | 2 +-
 man/io_prep_pread.3    | 2 +-
 man/io_prep_pwrite.3   | 2 +-
 man/io_queue_init.3    | 2 +-
 man/io_queue_release.3 | 2 +-
 man/io_queue_run.3     | 2 +-
 man/io_queue_wait.3    | 2 +-
 man/io_set_callback.3  | 2 +-
 man/io_submit.3        | 2 +-
 13 files changed, 13 insertions(+), 13 deletions(-)

diff --git a/man/io.3 b/man/io.3
index bc529ea..922fbca 100644
--- a/man/io.3
+++ b/man/io.3
@@ -1,4 +1,4 @@
-.TH io 3 2002-09-12 "Linux 2.4" Linux IO"
+.TH io 3 2019-07-23 "Linux" "Linux AIO"
 .SH NAME
 io \- Asynchronous IO
 .SH SYNOPSIS
diff --git a/man/io_cancel.3 b/man/io_cancel.3
index 261a407..a4683be 100644
--- a/man/io_cancel.3
+++ b/man/io_cancel.3
@@ -1,4 +1,4 @@
-.TH io_cancel 2 2002-09-03 "Linux 2.4" "Linux AIO"
+.TH io_cancel 3 2019-07-23 "Linux" "Linux AIO"
 .SH NAME
 io_cancel \- Cancel io requests
 .SH SYNOPSIS
diff --git a/man/io_fsync.3 b/man/io_fsync.3
index 5400d41..bc6778a 100644
--- a/man/io_fsync.3
+++ b/man/io_fsync.3
@@ -4,7 +4,7 @@
 .\" 	io_set_callback(iocb, cb);
 .\" 	return io_submit(ctx, 1, &iocb);
 .\" }
-.TH io_fsync 3 2002-09-12 "Linux 2.4" Linux AIO"
+.TH io_fsync 3 2019-07-23 "Linux" "Linux AIO"
 .SH NAME
 io_fsync \- Synchronize a file's complete in-core state with that on disk
 .SH SYNOPSIS
diff --git a/man/io_getevents.3 b/man/io_getevents.3
index 95fa86e..05b2507 100644
--- a/man/io_getevents.3
+++ b/man/io_getevents.3
@@ -16,7 +16,7 @@
 .\"                                 struct io_event *events,
 .\"                                 struct timespec *timeout)
 .\"
-.TH io_getevents 2 2002-09-03 "Linux 2.4" "Linux AIO"
+.TH io_getevents 3 2019-07-23 "Linux" "Linux AIO"
 .SH NAME
 io_getevents, aio_pgetevents \- Read resulting events from io requests
 .SH SYNOPSIS
diff --git a/man/io_prep_fsync.3 b/man/io_prep_fsync.3
index 0397172..bc0c43b 100644
--- a/man/io_prep_fsync.3
+++ b/man/io_prep_fsync.3
@@ -5,7 +5,7 @@
 .\" 	iocb->aio_lio_opcode = IO_CMD_FSYNC;
 .\" 	iocb->aio_reqprio = 0;
 .\" }
-.TH io_prep_fsync 3 2002-09-12 "Linux 2.4" Linux AIO"
+.TH io_prep_fsync 3 2019-07-23 "Linux" "Linux AIO"
 .SH NAME
 io_prep_fsync \- Synchronize a file's complete in-core state with that on disk
 .SH SYNOPSIS
diff --git a/man/io_prep_pread.3 b/man/io_prep_pread.3
index 3c28f61..e981e41 100644
--- a/man/io_prep_pread.3
+++ b/man/io_prep_pread.3
@@ -8,7 +8,7 @@
 .\" 	iocb->u.c.nbytes = count;
 .\" 	iocb->u.c.offset = offset;
 .\" }
-.TH io_prep_pread 3 2002-09-12 "Linux 2.4" Linux AIO"
+.TH io_prep_pread 3 2019-07-23 "Linux" "Linux AIO"
 .SH NAME
 io_prep_pread \- Set up asynchronous read
 .SH SYNOPSIS
diff --git a/man/io_prep_pwrite.3 b/man/io_prep_pwrite.3
index 69479c2..f2837b6 100644
--- a/man/io_prep_pwrite.3
+++ b/man/io_prep_pwrite.3
@@ -8,7 +8,7 @@
 .\" 	iocb->u.c.nbytes = count;
 .\" 	iocb->u.c.offset = offset;
 .\" }
-.TH io_prep_pwrite 3 2002-09-12 "Linux 2.4" Linux AIO"
+.TH io_prep_pwrite 3 2019-07-23 "Linux" "Linux AIO"
 .SH NAME
 io_prep_pwrite \- Set up iocb for asynchronous writes
 .SH SYNOPSIS
diff --git a/man/io_queue_init.3 b/man/io_queue_init.3
index 9c04b3f..d8753a7 100644
--- a/man/io_queue_init.3
+++ b/man/io_queue_init.3
@@ -1,4 +1,4 @@
-.TH io_queue_init 2 2002-09-03 "Linux 2.4" "Linux AIO"
+.TH io_queue_init 3 2019-07-23 "Linux" "Linux AIO"
 .SH NAME
 io_queue_init \- Initialize asynchronous io state machine
 
diff --git a/man/io_queue_release.3 b/man/io_queue_release.3
index 5c9ded1..392a478 100644
--- a/man/io_queue_release.3
+++ b/man/io_queue_release.3
@@ -1,4 +1,4 @@
-.TH io_queue_release 2 2002-09-03 "Linux 2.4" "Linux AIO"
+.TH io_queue_release 3 2019-07-23 "Linux" "Linux AIO"
 .SH NAME
 io_queue_release \- Release the context associated with the userspace handle
 .SH SYNOPSIS
diff --git a/man/io_queue_run.3 b/man/io_queue_run.3
index 8e0691c..dc54acd 100644
--- a/man/io_queue_run.3
+++ b/man/io_queue_run.3
@@ -1,4 +1,4 @@
-.TH io_queue_run 2 2002-09-03 "Linux 2.4" "Linux AIO"
+.TH io_queue_run 3 2019-07-23 "Linux" "Linux AIO"
 .SH NAME
 io_queue_run \- Handle completed io requests
 .SH SYNOPSIS
diff --git a/man/io_queue_wait.3 b/man/io_queue_wait.3
index 0cf0497..356c3b2 100644
--- a/man/io_queue_wait.3
+++ b/man/io_queue_wait.3
@@ -1,4 +1,4 @@
-.TH io_queue_wait 2 2002-09-03 "Linux 2.4" "Linux AIO"
+.TH io_queue_wait 3 2019-07-23 "Linux" "Linux AIO"
 .SH NAME
 io_queue_wait \- Wait for io requests to complete
 .SH SYNOPSIS
diff --git a/man/io_set_callback.3 b/man/io_set_callback.3
index 3599e8c..25b9a7f 100644
--- a/man/io_set_callback.3
+++ b/man/io_set_callback.3
@@ -1,5 +1,5 @@
 .\"static inline void io_set_callback(struct iocb *iocb, io_callback_t cb)
-.TH io_set_callback 3 2002-09-12 "Linux 2.4" Linux AIO"
+.TH io_set_callback 3 2019-07-23 "Linux" "Linux AIO"
 .SH NAME
 io_set_callback \- Set up io completion callback function
 .SH SYNOPSIS
diff --git a/man/io_submit.3 b/man/io_submit.3
index 3ae8721..01c25cd 100644
--- a/man/io_submit.3
+++ b/man/io_submit.3
@@ -10,7 +10,7 @@
 .\" *      are available to queue any iocbs.  Will return 0 if nr is 0.  Will
 .\" *      fail with -ENOSYS if not implemented.
 .\" */
-.TH io_submit 2 2002-09-02 "Linux 2.4" "Linux AIO"
+.TH io_submit 3 2019-07-23 "Linux" "Linux AIO"
 .SH NAME
 io_submit \- Submit io requests
 .SH SYNOPSIS
-- 
2.26.0.292.g33ef6b2f38

